qml/js/NetworkDeployment.js

```
@@ -279,8 +279,12 @@ function copyFiles(path, target)
 				continue
 			var deployDir = target + dirs[i].fileName + "/"
 			if (!fileIo.dirExists(deployDir))
```

This process now fails silently if the directory already exists.